ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConfigureConnectorTypeForm.tsx

```diff
@@ -56,58 +78,85 @@ export const ConfigureConnectorTypeForm: React.FunctionComponent<IConfigureConne
       basicValidationSchema[key.name] = Yup.string();
     }
     if (key.isMandatory) {
-      basicValidationSchema[key.name] = basicValidationSchema[key.name].required(`${key.displayName} is required`);
+      basicValidationSchema[key.name] = basicValidationSchema[
+        key.name
+      ].required(`${key.displayName} is required`);
     }
-  })
+  });
 
   const validationSchema = Yup.object().shape({ ...basicValidationSchema });
 
-  const toggle = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>, id: string) => {
+  const toggle = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    id: string
+  ) => {
     e.preventDefault();
     const index = expanded.indexOf(id);
     const newExpanded =
-      index >= 0 ? [...expanded.slice(0, index), ...expanded.slice(index + 1, expanded.length)] : [...expanded, id];
+      index >= 0
+        ? [
+            ...expanded.slice(0, index),
+            ...expanded.slice(index + 1, expanded.length),
+          ]
+        : [...expanded, id];
     setExpanded(newExpanded);
   };
-  
+
   const getInitialValues = (combined: any) => {
     const combinedValue: any = {};
-    
-    combined.map((key: { name: string; defaultValue: string}) => {
+
+    combined.map((key: { name: string; defaultValue: string }) => {
       if (!combinedValue[key.name]) {
         combinedValue[key.name] = key.defaultValue || "";
       }
-    })
+    });
     return combinedValue;
-  }
-  
+  };
+
   const handlePropertyChange = (propName: string, propValue: any) => {
-    propName = propName.replace(/\_/g, '.');
-    if(propName === PropertyName.PLUGIN_NAME) {
-      setShowPublication( propValue === "Pgoutput" );
+    propName = propName.replace(/\_/g, ".");
+    if (propName === PropertyName.PLUGIN_NAME) {
+      setShowPublication(propValue === "Pgoutput");
     }
-  }
+  };
+
+  const initialValues = getInitialValues(
+    _.union(
+      basicPropertyDefinitions,
+      advancedGeneralPropertyDefinitions,
+      advancedReplicationPropertyDefinitions,
+      advancedPublicationPropertyDefinitions
+    )
+  );
+
+  const handleSubmit = (valueMap: Map<string, string>) => {
+    // Validate the basic properties
```

Comment:
I think the basic and advanced properties could be combined - would result in just a single call to the backend.  wdyt?

Reply:
I separated as I thought we might handle(logically) the "valid"/"invalid" response differently to basic and advance property.

Reply:
I think all properties should be validated at once, but right, we may want to handle the response differently depending on whether the problem property is basic or advanced.

Reply:
yeah, we can check it with Gunnar & Team if they want the user to move ahead if only basic props are valid